src/commands/svc/enable.ts

```diff
@@ -3,7 +3,7 @@ import { BundleService } from '../../services/bundle-service'
 import { SvcService } from '../../services/svc-service'
 
 export default class Enable extends Command {
-  static description = 'Enable auxiliary services'
+  static description = 'Enables auxiliary services'
```

Comment:
As agreed, let's use imperative verbs

Reply:
Sorry @jeffgo10 for making you made the wrong changes. @cecchisandrone I missed that we agreed that format. We should also change the description of other commands (like init, pack, list, ...) since they use the other form.

Reply:
no worries @zonia3000, I already changed the description form in other commands like the ones you mentioned